### PR TITLE
Tpf 12 chat main

### PR DESCRIPTION
**Pull request by Jaydotcom27**
image

### CODE REVIEW

File: simple-chat-front/src/components/Chat/Chat-Style.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
+.chat{
+    width: 70%;
+    height: 75% ; 
+    display: flex;
+    flex-direction: column;
+}
+
+.chat__header{
```
Comment (rem1niscence) [inline simple-chat-front/src/components/Chat/Chat-Style.css:8]: the usual convention for CSS that I've seen is dashes not underscored but if that works no problem

File: simple-chat-front/src/App.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+import Chat from './components/Chat/Chat';
 // import TestComponent from './components/TestComponent/TestComponent'
 import UserNameInput from './components/UserNameInput/UserNameInput';
 
 function App() {
   return (
     <div className="App">
       <UserNameInput />
+      <Chat></Chat>
       {/* <header className="App-header">
```
Comment (rem1niscence) [inline simple-chat-front/src/App.js]: delete this unused comment

**Review by rem1niscence — CHANGES_REQUESTED** [review]

File: simple-chat-front/src/components/Chat/Chat-Style.css (hunk 1 above)
Comment (Jaydotcom27) [reply simple-chat-front/src/components/Chat/Chat-Style.css:8]: The convention we're using is BEM, check it out if you want to http://getbem.com/naming/

Comment (rem1niscence) [reply simple-chat-front/src/components/Chat/Chat-Style.css:8]: awesome, I'll do it, thanks

**Review by rem1niscence — APPROVED** [review]

**Pull request merged**